chainreg/chainregistry.go

```diff
@@ -256,46 +219,22 @@ func GenDefaultBtcConstraints() channeldb.ChannelConstraints {
 //
 //nolint:lll
 func NewPartialChainControl(cfg *Config) (*PartialChainControl, func(), error) {
-	// Set the RPC config from the "home" chain. Multi-chain isn't yet
-	// active, so we'll restrict usage to a particular chain for now.
 	homeChainConfig := cfg.Bitcoin
```

Comment:
We could probably get rid of this temporary variable entirely.

Reply:
removed 👍